var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -91,8 +91,10 @@ class Openmpi(AutotoolsPackage):
     variant(
         'fabrics',
         default=None if _verbs_dir() is None else 'verbs',
-        description='List of fabrics that are enabled',
-        values=('psm', 'psm2', 'pmi', 'verbs', 'mxm'),
+        description=("List of fabrics that are enabled ('rdma' enables "
+                     "'verbs', but implemented via the 'rdma-core' Spack "
+                     "package)"),
+        values=('psm', 'psm2', 'pmi', 'rdma', 'verbs', 'mxm'),
```

Comment:
This is a bit confusing (but maybe I don't read it right) - all these (with the exception of verbs) are separate APIs that are not part of rdma-core. Some of these don't use rdma-core at all.